frontend/services/entities/mdm_apple.ts

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import { sendRequest } from "services/mock_service/service/service"; // MDM TODO: Replace when backend is merged
+// import sendRequest from "services";
+import endpoints from "utilities/endpoints";
+
+export default {
+  loadAll: () => {
```

Comment:
this method's name doesn't match what it's doing. im not 100% certain but I think it's meant to get the apple MDM information and not loading multiple resources. Can we rename it to better reflect what the function is doing? same for mdm_apple_bm